logos/logos-workernode/logos_worker_node/logos_bridge.py

```
@@ -915,6 +924,17 @@ async def _run_calibration_session(self, session: _CalibrationSession) -> None:
                     existing[model_name] = new_profile
                     save_profiles(profiles_path, existing)
                     model_profiles._load_persisted()  # noqa: SLF001
+                    # Models that were pruned from capabilities at startup
+                    # because they had no profile must be re-announced now
+                    # that they're calibrated; otherwise the server never
+                    # learns the worker can serve them.
+                    if model_name not in self._cfg.capabilities_models:
+                        self._cfg.capabilities_models = list(self._cfg.capabilities_models) + [model_name]
```

Ruff (0.15.15)

[warning] Consider `[*list(self._cfg.capabilities_models), model_name]` instead of concatenation

Replace with `[*list(self._cfg.capabilities_models), model_name]`

(RUF005)